### PR TITLE
fix scoping for imagePullSecrets

### PR DESCRIPTION
**Pull request by ifrido**
#### What this PR does / why we need it:
Scoping for .Values.image.imagePullSecrets led to a templating error when providing secrets from values.

#### Which issue this PR fixes
--

#### Special notes for your reviewer:
--

#### Checklist
- [ ] Chart Version bumped

### CODE REVIEW

**Comment by zoomoid** [comment]
This one's on me... sorry for any inconvenience this might have caused

**Comment by fibis** [comment]
Can this please merged? We are getting the same error message when deploying zammad.

**Review by zoomoid — APPROVED** [review]
LGTM

**Review by monotek — CHANGES_REQUESTED** [review]
please raise chart version

**Comment by ifrido** [comment]
> please raise chart version

my bad, did not read the manual closely.
version bumped to 6.0.5

File: zammad/Chart.yaml [hunk 1]
```diff
@@ -17,17 +17,17 @@ maintainers:
 dependencies:
   - name: elasticsearch
     repository: https://helm.elastic.co
-    version: 7.16.2
+    version: 7.16.3
     condition: zammadConfig.elasticsearch.enabled
   - name: memcached
-    version: 5.15.5
+    version: 6.0.2
```
Comment (monotek) [inline zammad/Chart.yaml:23]: As this is a major version upgrade only raising the bugfix chart version seems a bit risky.
The braking changes of the memcached chart should be mentioned in the upgrading section of this chart too.

**Review by monotek — CHANGES_REQUESTED** [review]

**Pull request merged**